python/google/protobuf/internal/well_known_types_test.py

```diff
@@ -36,18 +36,18 @@
 
 from datetime import datetime
 
-try:
-  import unittest2 as unittest
-except ImportError:
-  import unittest
-
```

Comment:
Revert it:
https://github.com/calbach/protobuf/commits/python-json-struct/python/google/protobuf/internal/well_known_types_test.py